test/backend/remote.js

```diff
@@ -551,6 +567,81 @@ module.exports = function(cfg) {
   )
 
   test(
+    'get an unknown path',
+    function (t) {
+      testFailureEvent(t, serverOn, 2)
```

Comment:
I like not repeating yourself in each test, but something about the way this is called seems to obscure its intention.  I wonder if we could do something like:

```
t.plan(3)
failures = captureFailureEvents(server)
client.getThen('/foo').then(
  ...
).then( function(err) {
  testNotFound(t, err)
  t.equal(failures.length, 1, "emitted the failure event")
)}
```

What's nice about this is it keeps the `t.plan` and `t.equal` calls locally in the test, so you can get a sense of the entire flow of the test just by looking locally in that function.  Thoughts?